### PR TITLE
Finish client container logic

### PR DESCRIPTION
**Pull request by Z1Meme**

### CODE REVIEW

**Comment by Z1Meme** [comment]
fully tested and ready, however still can't move items into loadout slots (reverse issue)

File: src/servers/ZoneServer2016/commands/hax.ts [hunk 1]
```diff
@@ -644,12 +644,14 @@ const hax: any = {
       server.updateEquipmentSlot(client, 3);
     }
   },
+  /*
   addallitems: function (server: ZoneServer2016, client: Client, args: any[]) {
```
Comment (QuentinGruber) [inline src/servers/ZoneServer2016/commands/hax.ts:648]: so we disable this command ?

File: src/servers/ZoneServer2016/zonepackethandlers.ts [hunk 2]
```diff
@@ -278,9 +273,9 @@ export class zonePacketHandlers {
       client: Client,
       packet: any
     ) {
-      const characterId = packet.data.characterId;
-      const damage = packet.data.damage;
-      const vehicle = server._vehicles[characterId];
+      const characterId = packet.data.characterId,
+      damage = packet.data.damage,
```
Comment (QuentinGruber) [inline src/servers/ZoneServer2016/zonepackethandlers.ts:277]: i've to admit that i'm not a huge fan of this syntax, i will just do a quick bench to be sure if it doesn't change anything ( it shouldn't )

**Review by QuentinGruber — APPROVED** [review]
i will test this in-game and also run a benchmark to see if a particular function slow something :)

File: src/servers/ZoneServer2016/zonepackethandlers.ts (hunk 2 above)
Comment (QuentinGruber) [reply src/servers/ZoneServer2016/zonepackethandlers.ts:277]: yep it doesn't change anything the only bad thing is  that if you mess up a comma like that the variable become a global one ( which is never cleaned from memory) but as long at it never happen it's good 👍

File: src/servers/ZoneServer2016/classes/character.ts [hunk 3]
```diff
@@ -106,7 +106,7 @@ export class Character2016 extends Character {
         const { stamina, food, water, virus, health, bleeding } =
           client.character.resources;
         const { isRunning } = client.character;
-        if (isRunning) {
+        if (isRunning && (client.vehicle.mountedVehicle == "" || !client.vehicle.mountedVehicle)) {
```
Comment (QuentinGruber) [inline src/servers/ZoneServer2016/classes/character.ts:109]: i think it's not the good way to do it, like isRunning whould be equal to false when driving a vehicle

**Review by QuentinGruber — CHANGES_REQUESTED** [review]

File: src/servers/ZoneServer2016/classes/character.ts (hunk 3 above)
Comment (avcio9) [reply src/servers/ZoneServer2016/classes/character.ts:109]: Setting isRunning to false when entering vehicle doesnt work 100% of the time, sometimes server received position packet from client after mounting so it gets set back to true

**Review by QuentinGruber — APPROVED** [review]

**Pull request merged**